R/weight-streetnet.R

```diff
@@ -1,9 +1,64 @@
 #' Weight a street network according to a specified weighting profile.
 #'
-#' Weight (or re-weight) an \pkg{sf} or `SC` (`silicate`)-formatted OSM street
-#' network according to a named profile, selected from (foot, horse, wheelchair,
-#' bicycle, moped, motorcycle, motorcar, goods, hgv, psv), or a customized
-#' version derived from those.
+#' Weight (or re-weight) an \pkg{sf} or \pkg{silicate} *("SC") formatted OSM
+#' street network according to a named profile, selected from (foot, horse,
+#' wheelchair, bicycle, moped, motorcycle, motorcar, goods, hgv, psv), or a
+#' cusstomized version dervied from those.
+#'
+#' @details The structure of networks generated by this function is dependent
+#' on many aspects of the input network, and in particular on specific
+#' key-value pairs defined in the underlying OpenStreetMap (OSM) data.
+#'
+#' OSM keys which can be used to determine restrictions, and so may be added to
+#' the `keep_cols` argument, include:
```

Comment:
Currently, it looks like that the behaviour of `weight_streetnet()` depends on the following fields (access, bicycle, foot, ...) only when they are specified in the `keep_cols` argument, but this is not the case. Right? For example, the reprex included in #251 shows that the function creates one-way edges even when `keep_cols = NULL`.